user/views.py

```diff
@@ -4,15 +4,17 @@
 from rest_framework.authtoken.views import ObtainAuthToken
 from rest_framework.settings import api_settings
 
-from user.serializers import UserSerializer
+from user.serializers import UserSerializer, AuthTokenSerializer
 
 
 class CreateUserView(generics.CreateAPIView):
     serializer_class = UserSerializer
+    permission_classes = ()
```

Comment:
The `permission_classes` attribute in `CreateUserView` is set to an empty tuple, which is unnecessary since the default is already an empty list. Consider removing this line unless you intend to explicitly specify no permissions.